packages/react/vite.config.ts

```diff
@@ -0,0 +1,20 @@
+import { defineConfig } from "vite";
+import dts from "vite-plugin-dts";
+import react from "@vitejs/plugin-react-swc";
+
+// https://vite.dev/config/
+export default defineConfig({
+    plugins: [react(), dts({ tsconfigPath: "./tsconfig.lib.json" })],
+    build: {
+        emptyOutDir: true,
+        lib: {
+            entry: {
+                index: "src/index.ts",
+            },
+            formats: ["es"],
+        },
+        rollupOptions: {
+            external: ["react", "react-dom", "wagmi", "@tanstack/react-query"],
```

Comment:
Can we import the package.json here and automatically exclude all the `Object.keys(pkg.peerDependencies)`?